test/integration/dataContract/validateDataContractFactory.spec.js

```diff
@@ -35,6 +36,24 @@ describe('validateDataContractFactory', () => {
     validateDataContract = validateDataContractFactory(validator);
   });
 
+  describe('byte size limit', () => {
```

Comment:
I think we don't need additional describe here. Just move it to the bottom and call spec like "should return an invalid result if data contract byte size is bigger than 15 Kb"